src/dds/rs-dds-sensor-proxy.cpp

```diff
@@ -144,6 +144,7 @@ void dds_sensor_proxy::register_basic_converters()
     _formats_converter.register_converter( processing_block_factory::create_id_pbf( RS2_FORMAT_RGBA8, RS2_STREAM_COLOR ) );
     _formats_converter.register_converter( processing_block_factory::create_id_pbf( RS2_FORMAT_BGR8, RS2_STREAM_COLOR ) );
     _formats_converter.register_converter( processing_block_factory::create_id_pbf( RS2_FORMAT_BGRA8, RS2_STREAM_COLOR ) );
+    _formats_converter.register_converter( processing_block_factory::create_id_pbf( RS2_FORMAT_RAW16, RS2_STREAM_COLOR ) );
```

Comment:
Is it a general stream that will serve for multiple camera models? Do we need to limit to a specific model?

Reply:
General: if a camera wants to expose RAW16, we let it through.